src/mscorlib/src/System/String.Comparison.cs

```diff
@@ -792,6 +792,25 @@ public int CompareTo(String strB)
             return false;
         }
 
+        [Pure]
+        public Boolean EndsWith(Char value)
+        {
+            return EndsWith(new string(value, 1), StringComparison.CurrentCulture);
```

Comment:
If these overloads aren't needed, we could reuse the implementation of the internal method EndsWithOrdinal(char)..